openapi/index.json

```diff
@@ -0,0 +1,47 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "version": "9.0.0",
+    "title": "GitHub v3 REST API",
+    "description": "# Introduction\n\nHaxx0r ipsum fatal dereference echo boolean fopen void mutex ctl-c wombat overflow.",
+    "license": {
+      "name": "MIT"
+    },
+    "termsOfService": "https://help.github.com/articles/github-terms-of-service/",
+    "contact": {
+      "name": "Support",
+      "email": "support@github.com"
+    },
+    "x-octokit-info": {
+      "debuggerPropertiesBySchema": {
+        "user": "id,login",
+        "label-summary": "id,name",
+        "milestone": "number,title",
+        "pull-request-summary": "",
+        "issue": "id,number"
+      }
+    }
```

Comment:
@ryangribble could you add an explanation on how you use that property for octokit.net?

Reply:
In the `Issue.cs` response model, we use it to generate this property:

`internal string DebuggerDisplay => string.Format(CultureInfo.InvariantCulture, "Issue Id: {Id}, Number: {Number}");`

Reply:
for each response object (schema) we need to know what are the "important" fields that should be shown when this object is being shown (eg in debugger window or mouse over)

Reply:
there was no suitable field on the actual `properties` openAPI entry, so I went down the path of using OpenAPI extensions.  They can only apply to certain places though, and under `schemas` isnt one of those... so although its not ideal to have them at the top `info` level, there wasn't a more suitable place really.

If the goal is to generate everything from "the source" then I think we will need to end up adding certain additional info into the openAPI description, because maintaining something like this on the octokit client side means a manual update whenever a new response schema is added

Reply:
Maybe the "important" fields are the URL parameters in general? So e.g. for [`GET /repos/:owner/:repo/issues/:number`](https://developer.github.com/v3/issues/#get-a-single-issue) it would be `["owner", "repo", "number"]`?

Reply:
It's a mismatch though, because these objects are declared in schemas area, so there is no url/parameters there